config/crd/bases/notification.toolkit.fluxcd.io_receivers.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.11.1
+    controller-gen.kubebuilder.io/version: v0.8.0
```

Comment:
Please delete the controller-gen binary and rerun make to use the latest version